src/keyDerivation.c

```diff
@@ -185,33 +184,30 @@ void addressRootFromExtPubKey(
 
 	WRITE_TOKEN(ptr, end, CBOR_TYPE_MAP, 0);
 
-	// cborBuf is hashed twice. First by sha3_256 and then by blake2b_224
+	// cborBuffer is hashed twice. First by sha3_256 and then by blake2b_224
 	uint8_t cborShaHash[32];
 	sha3_256_hash(
-	        cborBuf, ptr - cborBuf,
+	        cborBuffer, ptr - cborBuffer,
```

Comment:
I wonder -- should we create `REMAINING(ptr, end)` or even `REMAINING(ptr, baseBuffer)` macros?